File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Feedbacks/IFeedbackData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Feedbacks;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IFeedbackData extends IFeedbackTypes,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}

File: Ohtap E Commerce Domain Models/Development/NetCore/Models Objects/Models Objects/Feedbacks/IFeedbackData.cs
using Ohtap.ECommerce.Domain.Models.Objects.Datasets;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Feedbacks
{
    public interface IFeedbackData :  CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IFeedbackTypes
    {
        IDataItemList<IDataItemData> Dataset { get; set; }
    }
}
